### ISSUE TITLE
Synergy 1 Basic on Gnome Wayland

### ISSUE THREAD

**Issue by vdegenne** [symless/synergy-core#6610]
Here I've tried to make it work on my gnome desktop for about 3 hours, trying different configuration and tricks. I try to make it work with a Windows 10 aside.
But nothing's doing. I'm losing hope.
I guess this is because it's wayland, the reason I think that's because if I run on Xorg it works with no pain..
No wayland support ? if not is it something you're working on ? I guess wayland tends to unify things and I noticed it's much more faster than Xorg soo I guess this is a route you should consider to take about further development and I should consider to purchase the Pro version.

A regular user.

**Comment by vdegenne** [comment]
wait... now it works. just after writing this post.. I don't understand. well.. thanks for the fast support.

**Comment by vdegenne** [comment]
I reopened because I noticed something strange. After I wrote this post it worked then I investigated.
It only works on some windows, for instance if I open a terminal (alacritty, gnome, etc...) it doesn't work anymore I try to push the cursor out to the right but it just blocks on the far most of the screen.
On the other hand if I activate let say Chrome window and try to navigate the cursor on the other screen on the right it just works. Anyway to fix that ?
